serializer/serializer_test.go

```diff
@@ -87,8 +86,7 @@ func TestGetSerializedDataMarshalEmptyCache(t *testing.T) {
 	result, err := serializer.GetSerializedData()
 
 	// Validate that returned logging script contains a valid SplitData
-	expectedData := "{}"
-	expectedLoggingScript := fmt.Sprintf(formattedLoggingScript, expectedData)
+        expectedLoggingScript := "\n<script>\n  window.__splitCachePreload = {\n  splitsData: null,\n  since: 0,\n  segmentsData: null,\n  usingSegmentsCount: 0\n  };\n</script>"
```

Comment:
I think node version made a change to return "{}" in this case and we want to do the same thing for Go?
https://github.com/godaddy/split-node-serializer/pull/27

Reply:
yup, i'll do that in a follow up pr